data/mods/Medieval_Stuff/armor.json

```diff
@@ -0,0 +1,294 @@
+        "color" : "dark_gray",
+        "covers" : ["LEGS", "TORSO", "ARMS"],
+        "to_hit" : -5,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A simple garment long enough to cover down to the knees.",
+        "price" : 4200,
+        "material" : ["cotton", "null"],
+        "volume" : 5,
+        "cutting" : 0,
+        "warmth" : 2,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 0,
+        "bashing" : -5,
+        "flags" : ["VARSIZE"],
+        "coverage" : 75,
+        "material_thickness" : 1
+    },{
+        "type" : "ARMOR",
+        "id" : "gambeson",
+        "name" : "gambeson",
+        "weight" : 453,
+        "color" : "light_gray",
+        "covers" : ["TORSO", "ARMS", "LEGS"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A thick jacket of quited fabric, designed to be worn underneath mail or other armor.  Or worn on its own, if you can't afford proper armor.",
+        "price" : 20000,
+        "material" : ["cotton", "null"],
+        "volume" : 8,
+        "cutting" : 0,
+        "warmth" : 30,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 10,
+        "bashing" : -2,
+        "flags" : ["VARSIZE", "SKINTIGHT", "STURDY"],
+        "coverage" : 80,
+        "material_thickness" : 5
+    },{
+        "type" : "ARMOR",
+        "id" : "armor_cuirass",
+        "name" : "bell cuirass",
+        "name_plural" : "bell cuirasses",
+        "weight" : 5896,
+        "color" : "yellow",
+        "covers" : ["TORSO"],
+        "to_hit" : -5,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "An ancient Greek breastplate, made of bronze.",
+        "price" : 50000,
+        "material" : ["bronze", "null"],
+        "volume" : 24,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : 6,
+        "flags" : ["VARSIZE", "OUTER", "STURDY"],
+        "coverage" : 90,
+        "material_thickness" : 3
+    },{
+        "type" : "ARMOR",
+        "id" : "armor_lorica",
+        "name" : "lorica segmentata",
+        "name_plural" : "lorica segmentata",
+        "weight" : 5286,
+        "color" : "light_gray",
+        "covers" : ["TORSO", "ARMS"],
+        "to_hit" : -5,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "An ancient Roman laminar armor, made of overlapping metal strips connected to internal leather straps.",
+        "price" : 50000,
+        "material" : ["iron", "leather"],
+        "volume" : 24,
+        "cutting" : 0,
+        "warmth" : 15,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 15,
+        "bashing" : 6,
+        "flags" : ["VARSIZE", "OUTER", "STURDY"],
+        "coverage" : 90,
+        "material_thickness" : 4
```

Comment:
Same armor value as plate armor, with less than half the encumbrance. Has lower coverage, but still.